packages/linux/package.mk

```diff
@@ -148,11 +150,16 @@ pre_make_target() {
     # copy some extra firmware to linux tree
     mkdir -p $PKG_BUILD/external-firmware
       cp -a $(get_build_dir kernel-firmware)/.copied-firmware/{amdgpu,amd-ucode,i915,radeon,e100,rtl_nic} $PKG_BUILD/external-firmware
+      cp -a $(get_build_dir intel-ucode)/intel-ucode $PKG_BUILD/external-firmware
+  elif [ "$TARGET_ARCH" = "arm" -a "$DEVICE" = "iMX6" ]; then
+    mkdir -p $PKG_BUILD/external-firmware
+      cp -a $(get_build_dir kernel-firmware)/imx $PKG_BUILD/external-firmware
```

Comment:
why not use the sdma firmware from the firmware-imx package? you might as well build in all the firmware then.

Reply:
Yes, maybe firmware-imx will do better here. I'll think about it.